src/styles/custom.css

```diff
@@ -9,3 +9,8 @@
 		grid-template-columns: 1fr 1fr;
 	}
 }
+
+/* Hide the guideline for the final step in <Steps> lists. */
+.sl-steps > li:last-of-type::after {
+  background: transparent;
+}
```

Comment:
I think we can use the new Starlight styles directly instead of overriding them. This style was changed to make the last step in `<Steps>` more obvious to users based on user feedback, so I think the new Starlight default makes sense for us in Astro Docs too.

```suggestion
```